webapp/src/components/properties/link/link.scss

```diff
@@ -0,0 +1,26 @@
+.property-link {
+    &.url {
+        padding-right: 2rem;
+    }
+
+    a {
+        padding: 8px; // increases clickable area for better UX
+        display: none;
```

Comment:
I think the `display: none` should be removed and just show the icon all the time for link property.

Reply:
I know this means they can enter a invalid URL and get redirected by clicking the icon but atleast we have a red broder around the link property indicating it's invalid and I feel at that point it's on them.

Reply:
Keeping the icon hidden until hovered on, so need to preserve `display: none`.